src/clone.rs

```diff
@@ -169,6 +190,23 @@ macro_rules! to_return_value {
 /// closure is immediately returning an optional default return value. If none is provided, `()` is
 /// returned.
 ///
+/// ### Debugging
+///
+/// In case something goes wrong inside the `clone!` macro, we use the [`g_debug`] macro. Meaning
+/// that if you want to see these debug messages, you'll have to set the `G_MESSAGES_DEBUG`
```

Comment:
set to what? :) Don't you need `G_MESSAGES_DEBUG=all` or `G_MESSAGES_DEBUG=glib-rs-clone` or so? The latter for only having logs from this specific domain.

Reply:
I used `G_MESSAGES_DEBUG=1` and it worked so I don't know? XD

Reply:
I wrote how just below already. :p

Reply:
No, you didn't say which log domain it is using. And `G_MESSAGES_DEBUG` takes a string (comma separated listed of domains) or `"all"` for everything. Not `"1"`